astropy/utils/introspection.py

```diff
@@ -255,7 +255,7 @@ def find():
                 if inspect.ismodule(fd):
                     diffmods.append(fd)
                 elif isinstance(fd, str):
-                    diffmods.append(__import__(fd))
+                    diffmods.append(importlib.import_module(fd))
```

Comment:
TBH, I think this *may* have been a feature at the time, not a bug.  But I don't think I know of any cases where this is used for anything other than the top-level package?  (for which the `__import__` and `import_module` behavior are the same.

The one place I'm worried about is the sphinx machinery.  Checking that now.  But regardless it's probably fair to call this a bug fix rather than an API change: the docs pretty clearly use the term "module".  I'm only worried that someone might have written some code assuming the ``__import__`` behavior. I don't see anything on github's code search though, so probably in the clear?

Reply:
Seems OK - the before- and after- sphinx docs build seems identical, so good to go.